recipes/rapid/v0.8/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "0.8" %}
+
+package:
+  name: rapid
+  version: {{ version }}
+
+source:
+  url: https://github.com/SchulzLab/RAPID/archive/v{{ version }}.tar.gz
+  sha256: caddaa79d9be8012bc1ab09bdc431b863c80414eb471ac3a79ce5c940a03d4b5
+build:
+  number: 0
+
+requirements:
+  build:
+    - r-base >=3.4.1
```

Comment:
Please remove all the pinnings if not really needed.
Also with conda-build 3, this should be the `host` section and not the build section.

So you really need perl in the host section?

Reply:
Thanks for pointing out @bgruening . I modified the section to `host`, and removed Perl. 
I tried building locally, and unfortunately, I still face the same issue. 

And, I would prefer to have the other pinnings, in order to avoid issues from deprecated functionalities/incompatabilities across different versions.

Reply:
I removed all the pinning, and it successfully passed the tests. Thanks @bgruening .
 Now will test it out if the functionalities of my pipeline works fine.